File: tests/compatibility/v0.5.3/csharp/compatibility.cs
using Uimd;

public class Compatibility : CompatibilityUI
{
    // Override generated event hooks here.
}

public static class Program
{
    public static int Main(string[] args)
    {
        return GeneratedWindowRuntime.RunGeneratedAppMain(() =>
        {
            Compatibility app = new();
            return GeneratedWindowRuntime.RunGeneratedWindow(app, app.RuntimeOptions(), args);
        });
    }
}

File: tests/compatibility/v0.5.3/cpp/compatibility.cpp
// Application logic for compatibility.
#include "compatibility_ui.hpp"

#include "ui/generated/GeneratedWindowRuntime.hpp"

#include <string>

class Compatibility : public CompatibilityUI
{
protected:
    // Override generated event hooks here.

};

int main(int argc, char** argv)
{
    return ui::runGeneratedAppMain([&]() -> int
    {
        Compatibility app;
        return ui::runGeneratedWindow(app, app.runtimeOptions(), argc, argv);
    });
}
